src/extension.ts

```diff
@@ -420,6 +422,13 @@ class RdbgAdapterDescriptorFactory implements DebugAdapterDescriptorFactory {
 
 			if (outputTerminal) {
 				outputTerminal.show(false);
+
+				if (config.cwd) {
+					// Ensure we are in the requested working directory
+					const cd_command = "cd " + config.cwd;
+					outputTerminal.sendText(cd_command);
```

Comment:
Does it work correctly if `cwd` is relative path?

Reply:
It would need to be a relative path from the workspace root.

Reply:
Is it safe to start repeatedly?

Reply:
I tried with `cwd: "foo"` (there is `${workspaceFolder}/foo` directory) and it doesn't work on line `get_sock_list()` (ENOENT), so my question "Does it work correctly if cwd is relative path?" is "NO" and my real question "does it work on repeated launch with relative path cwd" is YES because this code is not reachable with a relative path.

(FYI: `"cwd": "${workspaceFolder}/foo"` works fine)

For better solution maybe we need to use `File.expand_path()` in Ruby to solve the relative path to absolute path, but I'm not sure how to do it on nodejs.

I'm okay to merge this commit. Do you want to care about this `expand_path` or can I merge it.

Reply:
> My env also uses rvm, which requires some special handling. I have the following file in ${workspaceFolder}/.vscode/rdbg:

Is it relative to this line or this PR?
Should we discuss on other PR?

Reply:
In other words, can I merge it and close it?

Reply:
Yes, this PR can be merged and closed.

I wanted to see if you're interested in documenting the above behavior. If you are then I can open another PR for that.

Reply:
Thank you. I can't understand your point. To use with rvm this debugger needs wrapper script, you mean?

Reply:
Correct. I just want to know if you would like me to make a PR describing how I did this in the README.md.

Reply:
Before adding it to the doc, I want to know why this wrapper is needed for rvm.
But this discussion should be done in other issue/PR, maybe.